main/input_default.cpp

```diff
@@ -580,6 +580,15 @@ void InputDefault::action_release(const StringName &p_action) {
 	action_state[p_action] = action;
 }
 
+void InputDefault::action_release_all() {
+
+	for (Map<StringName, Action>::Element *E = action_state.front(); E; E = E->next()) {
+		if (E->get().pressed) {
+			action_release(E->key());
+		}
+	}
+}
```

Comment:
```suggestion
	keys_pressed.clear();
}
```
If we also clear keys_pressed, PR can fix #22265, WDYT?

Reply:
Sounds good, just added it, thanks!

Reply:
Shouldn't it clear `joy_buttons_pressed` and `_joy_axis` in this case as well?

Reply:
Just changed it to `release_all` and added the clears for `joy_buttons_pressed` and `_joy_axis`